pkg/manifest/operations.go

```diff
@@ -297,7 +297,14 @@ func (o *Operations) uninstall(deployInfo types.InstallInfo) (bool, error) {
 	return true, err
 }
 
-func (o *Operations) getManifestForChartPath(installInfo types.InstallInfo) *types.ParsedFile {
+func (o *Operations) GetManifestForChartPath(installInfo types.InstallInfo) *types.ParsedFile {
```

Comment:
Is it necessary to export this method from Operations?

Reply:
agree, not necessary, I convert it back.

Reply:
not necessary, only exposed for `operations_test.go`